bin/cqlsh.py

```diff
@@ -600,6 +601,12 @@ def show_version(self):
     def show_session(self, sessionid, partial_session=False):
         print_trace_session(self, self.session, sessionid, partial_session)
 
+    def show_replicas(self, token, keyspace=None):
+        ks = self.current_keyspace if keyspace is None else keyspace
+        nodes = self.conn.metadata.token_map.get_replicas(self.current_keyspace, Murmur3Token(token))
```

Comment:
should not this be `ks` instead of `self.current_keyspace` in `get_replicas` method?

Reply:
Yes, I'll fix that. Thanks!